docs/content/docs/get-started/installation.md

```diff
@@ -22,47 +22,46 @@ specific language governing permissions and limitations
 under the License.
 -->
 
-# Installation
-
+## Overview
 Flink Agents provides both Python and Java APIs to define a Flink Agents job.
 
-To try Flink Agents in Python on a local executor, you only need to install the Flink Agents Python package.
-
-To define a Flink Agents job using the Java API, or to run the job in a Flink cluster, you need to install both the Flink Agents Python and Java dependencies. 
+To try Flink-Agents in Python or Java, user can only install the correspond package.
 
 The sections below show how to install the required dependencies.
 
-## Install from PyPI
+{{< hint warning >}}
+__NOTE:__ To run on flink cluster, Flink-Agents requires flink version be a stable release of Flink 1.20.3.
+{{< /hint >}}
+## Install the Official Release
+
+#### Install Python Packages
 
 {{< hint warning >}}
 __Note:__ This will be available after Flink Agents is released.
 {{< /hint >}}
 
+We recommand creating a Python virtual environment to install the Flink Agents Python library.
+
 To install the latest Flink Agents release, run:
 
 ```shell
 python -m pip install flink-agents
 ```
 
-<!-- TODO: link to local quickstart example docs -->
-Now you can run a Flink Agents job on the local executor.
-See [local quickstart example]() for end-to-end examples of running on the local executor.
-
+Now you can run a python Flink Agents job.
+See [quickstart example]({{< ref "docs/get-started/quickstart/workflow_agent" >}}) for end-to-end examples of running on Flink.
```

Comment:
If we are telling users to always install both packages, we probably want to remove these paragraph, or move it to somewhere else.